readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -54,24 +54,25 @@ def append_conf(self, **kwargs):
 
         # Set mkdocs config values
 
-        media_url = getattr(settings, 'MEDIA_URL', 'https://media.readthedocs.org')
+        media_url = getattr(settings, 'MEDIA_URL', '')
```

Comment:
Thanks! Since Django set the default value of `MEDIA_URL` implicitly we can now change this line to 

```py
media_url = settings.MEDIA_URL
```